catalogs/catalog-hive/src/main/java/com/datastrato/gravitino/catalog/hive/HiveCatalogOperations.java

```diff
@@ -1088,6 +1088,12 @@ public PropertiesMetadata filesetPropertiesMetadata() throws UnsupportedOperatio
         "Hive catalog does not support fileset properties metadata");
   }
 
+  @Override
+  public PropertiesMetadata topicPropertiesMetadata() throws UnsupportedOperationException {
+    throw new UnsupportedOperationException(
+        "Hive catalog does not support topic properties metadata");
+  }
+
```

Comment:
Why not provide a default implementation instead of implementing a throw UnsupportedOperationException in each class?

Reply:
Because we need to draw the attention of the developers of the catalog to this interface.

Reply:
I think this will cause trouble for extensions. For example, if another xxxPropertiesMetadata is added to the interface, all subclasses will have to add the same code, isn't that troublesome?

Reply:
Maybe you could create a separate issue for this? Since `tablePropertiesMetadata` and `filesetPropertiesMetadata` have the same implementations, we should incorporate overall changes in another PR. 
What would you suggest? @jerryshao

Reply:
Yes, I think we can create a separate issue to rethink the current interface of `xxxPropertiesMetadata`.